lambdas/services/lloyd_george_stitch_service.py

```diff
@@ -55,23 +55,21 @@ def stitch_lloyd_george_record(self, nhs_number: str) -> str:
             )
 
         try:
-            filename_for_stitched_file = self.make_filename_for_stitched_file(
-                lg_records
-            )
             stitched_lg_record = stitch_pdf(all_lg_parts, self.temp_folder)
+            filename_for_stitched_file = os.path.basename(stitched_lg_record)
```

Comment:
Does this not still contain the patient's name?
I've got a couple of checks on this one....

- Do we reload the same file more than once or do we recreate the link every time? if We do reload the file more than once, should we hash the filename so we can generate it predictably?

Reply:
Discussed, this will be an improvement down the line once we can monitor LG last upload state in code.